manifests/plugin/ansible.pp

```diff
@@ -29,6 +29,7 @@
   Optional[Stdlib::Absolutepath] $working_dir = $::foreman_proxy::plugin::ansible::params::working_dir,
   Boolean $host_key_checking = $::foreman_proxy::plugin::ansible::params::host_key_checking,
   String $stdout_callback = $::foreman_proxy::plugin::ansible::params::stdout_callback,
+  Boolean $upstream_runner_repo = $::foreman_proxy::plugin::ansible::params::upstream_runner_repo,
```

Comment:
Is it necessary here?

Reply:
This exposes it to kafo and users. A more conventional naming is `manage_runner_repo`

Reply:
I've renamed the param